app/controllers/api/v1/posters_controller.rb

```diff
@@ -1,11 +1,20 @@
 class Api::V1::PostersController < ApplicationController
 
   def index
-    sort = params[:sort]
-    posters = Poster.sort_results(sort)
-		options = {}
-		options[:meta] = { count: Poster.count }
-    render json: PosterSerializer.new(posters, options)
+    posters =
```

Comment:
Good business logic in the controller, long, but clean. I'm curious how this may work as a refactor.